### DISCUSSION TITLE
Is it possible to create a badge for a related group of packages ? (using a wildcard query)

### DISCUSSION THREAD

**Discussion by Josverl** [psincraian/pepy#547] [Q&A] [answered]
I would like to create a batch that contains the combined/aggregated metrics of a family of packages 

In my case this is regarding stub packages for MicroPython 

there are multiple packages for the different ports and boards published :

- micropython-esp32-stubs
- micropython-esp32-ota-stubs
- micropython-esp32-s3stubs
and similar for the rp2 
- micropython-rp2-stubs
- micropython-rp2-pico-stubs
- micropython-rp2-picow-stubs
and other ports & boards 

It would be super nice if it would be possible to aggregate per port by using a wildcard ( * / % / .* or similar) 

https://pepy.tech/project/micropython-esp32-.*stubs
https://pepy.tech/project/micropython-rp2-.*stubs
https://pepy.tech/project/micropython-stm32-.*stubs
https://pepy.tech/project/micropython-samd-.*stubs

op perhaps even 
https://pepy.tech/project/micropython-.*-stubs

[![Downloads](https://static.pepy.tech/personalized-badge/micropython-esp32-stubs?period=month&units=international_system&left_color=black&right_color=orange&left_text=micropython-esp32-*-stubs)](https://pepy.tech/project/micropython-esp32-stubs)

I realize that there should be an acceptable upper limit to the projects to avoid abuse

**Comment by psincraian** [comment, marked as answer]
We don't have that feature currently, but I will keep open this discussion to know the amount of interest in this :)
